routers/web/repo/setting.go

```diff
@@ -911,6 +911,19 @@ func CollaborationPost(ctx *context.Context) {
 		return
 	}
 
+	// find the owner team of the organization the repo belongs too and
+	// check if the user we're trying to add is an owner.
+	if ctx.Repo.Repository.Owner.IsOrganization() {
+		if isOwner, err := organization.IsOrganizationOwner(ctx, ctx.Repo.Repository.Owner.ID, u.ID); err != nil {
+			ctx.ServerError("IsOrganizationOwner", err)
+			return
+		} else if isOwner {
+			ctx.Flash.Error(ctx.Tr("settings.add_collaborator_owner"))
```

Comment:
IIRC, it should has a `repo.` prefix because the locale key is in a section.

Otherwise L-G-T-M.

Reply:
good catch, fixed.